### PR TITLE
[SD-5076] - De-normalizing user and item in activity

### PR DESCRIPTION
**Pull request by akintolga**
Also added user_id to move and copy notifications...

### CODE REVIEW

**Comment by mdhaman** [comment]
:+1:

**Comment by marwoodandrew** [comment]
:+1:

File: apps/duplication/archive_copy.py [hunk 1]
```diff
@@ -19,7 +19,8 @@
 from superdesk.metadata.utils import item_url
 from apps.archive.archive import SOURCE as ARCHIVE
 from superdesk.workflow import is_workflow_state_transition_valid
-
+from eve.utils import config
+from apps.auth import get_user
```
Comment (petrjasek) [inline apps/duplication/archive_copy.py]: fyi there is `apps.auth.get_user_id` function

**Comment by petrjasek** [comment]
:+1:

**Comment by mdhaman** [comment]
@petrjasek will merge this PR. Will update the code as per your suggestion in next PR.

**Pull request merged**